package com.sxt.order.exception;

@SuppressWarnings("serial")
/**
 * 自定义异常类, 当用户支付订单, 但是订单状态为"已支付", 抛出此异常
 * @author robbyzhan
 *
 */
public class PayException extends Exception {

	public PayException() {
		super();
		// TODO Auto-generated constructor stub
	}

	public PayException(String message, Throwable cause) {
		super(message, cause);
		// TODO Auto-generated constructor stub
	}

	public PayException(String message) {
		super(message);
		// TODO Auto-generated constructor stub
	}

	public PayException(Throwable cause) {
		super(cause);
		// TODO Auto-generated constructor stub
	}
	
}
